AV-101448 - Documentation update all datasources (#328)
couchbasecloud/terraform-provider-couchbase-capella: docs/data-sources/projects.md

page_title: "couchbase-capella_projects Data Source - terraform-provider-couchbase-capella"
subcategory: ""
description: |-

>   Data source to retrieve projects' information in a Capella organization.

Reviewer: ```suggestion
  Data source to retrieve project details in an organization.
```

---

# couchbase-capella_projects (Data Source)

> Data source to retrieve projects' information in a Capella organization.

Reviewer: ```suggestion
Data source to retrieve project details in an organization.
```

### Required

> - `organization_id` (String) The GUID4 ID of the Capella organization.

Reviewer: ```suggestion
- `organization_id` (String) The GUID4 ID of the organization.
```
